README.md

```diff
@@ -337,7 +340,7 @@ func reticulateSplines(spline: [Double], adjustmentFactor: Double,
 Use trailing closure syntax wherever possible. In all cases, give the closure parameters descriptive names:
 
 ```swift
-return SKAction.customActionWithDuration(effect.duration) { node, elapsedTime in
+return SKAction.customAction(withDuration: effect.duration) { node, elapsedTime in
```

Comment:
Nemelo by to spis byt "SKAction.customAction(with duration: effect.duration) {..." ?